src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -51,115 +47,28 @@
-                .initFields(fields)
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewFlattened(context, rootNode);
-  }
-
-  public InitCodeView generateRequestObjectInitCode(MethodTransformerContext context) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(new SymbolTable())
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewRequestObject(context, rootNode);
-  }
-
-  public InitCodeView generateFlatteningTestInitCode(
-      MethodTransformerContext context,
-      Iterable<Field> fields,
-      SymbolTable table,
-      TestValueGenerator valueGenerator) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(table)
-                .valueGenerator(valueGenerator)
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .initFields(fields)
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewFlattened(context, rootNode);
-  }
-
-  public InitCodeView generateRequestObjectTestInitCode(
-      MethodTransformerContext context, SymbolTable table, TestValueGenerator valueGenerator) {
-    InitCodeNode rootNode =
-        InitCodeNode.createTree(
-            InitTreeParserContext.newBuilder()
-                .table(table)
-                .valueGenerator(valueGenerator)
-                .rootObjectType(context.getMethod().getInputType())
-                .initValueConfigMap(createCollectionMap(context))
-                .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
-                .suggestedName(Name.from("request"))
-                .build());
-    return buildInitCodeViewRequestObject(context, rootNode);
-  }
-
-  public InitCodeView generateMockResponseObjectInitCode(
-      MethodTransformerContext context, SymbolTable table, TestValueGenerator valueGenerator) {
-    ArrayList<Field> primitiveFields = new ArrayList<>();
-    for (Field field : context.getMethod().getOutputMessage().getFields()) {
-      if (field.getType().isPrimitive() && !field.getType().isRepeated()) {
-        primitiveFields.add(field);
-      }
+  /** Generates initialization code from the given InitCodeContext object. */
+  public static InitCodeView generateInitCode(InitCodeContext context) {
```

Comment:
I want the classes to work consistently. This would be the only transformer class with static methods. Either they should all have static methods, or they should all have instance methods. I don't think it is something you should change under the scope of this current review.